go.mod

```diff
@@ -135,11 +135,12 @@ replace github.com/99designs/keyring => github.com/cosmos/keyring v1.1.7-0.20210
 // TODO Remove it: https://github.com/cosmos/cosmos-sdk/issues/10409
 replace github.com/gin-gonic/gin => github.com/gin-gonic/gin v1.7.0
 
-// Fixes chain panic.  Should be removed once cosmwasm has fixed their release
-// Dockerfile also needs a reference that will need removing
-// TODO Remove it: https://github.com/CosmWasm/cosmwasm/pull/1223
-replace github.com/CosmWasm/wasmvm => github.com/CosmWasm/wasmvm v1.0.0-beta7
+// Introduces a way to build for aarch64 and includes a prebuilt shared library for it
```

Comment:
Do you want to keep the old comment as well so we know why it is in

Reply:
Sure, I can add them back in. Good catch.